lib/commands/start.js

```diff
@@ -45,6 +46,13 @@ module.exports = BaseCommand.extend({
         // TODO: rethink this
         return this.runCommand('config', 'paths.contentPath', path.join(process.cwd(), 'content'), {environment: environment})
             .then(function () {
+                knexMigrator = new KnexMigrator();
```

Comment:
I know this is still a WIP - but in order to improve load times of the CLI, there's a better way to have the requires work:

```js
var KnexMigrator = require('knex-migrator'),
    knexMigrator = new KnexMigrator();

// ... do other things
```

If you do this and remove the two changes above this then the code changes look good to me - will test it locally once it's out of WIP status :smile: